localization/ndt_scan_matcher/config/ndt_scan_matcher.param.yaml

```diff
@@ -1,5 +1,7 @@
 /**:
   ros__parameters:
+    # Use dynamic map loading
+    use_dynamic_map_loading: false
```

Comment:
I think the default value can be true.

Reply:
@yukkysaito Thank you! That may be a good idea. The actual performance wouldn't change as long as you provide a single pointcloud map even if `use_dynamic_map_loading` is set true. (The only minor drawback AFAIK is that NDT will still request for a new map for every 20 meters in vain)

But then I also think that we can even completely remove the previous map interface feature from NDT, which is left as it is in the current PR. What do you think? @YamatoAndo @KYabuuchi @yukkysaito

Reply:
(It is also OK for me to discuss about completely remove the previous map interface after merging the current PR, maybe somewhere like Autoware Discussion)

Reply:
OK, then I will just make the default value true.

Reply:
Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2339/commits/67585f8ceec126271389e102c91b06903fa0a42d